lib/ts/fetch.ts

```diff
@@ -926,3 +927,38 @@ export function fireSessionUpdateEventsIfNecessary(
         });
     }
 }
+
+/**
+ * Updates the clock skew based on the provided frontToken and theround-trip time.
+ *
+ * @param {Object} params - The parameters for updating the clock skew.
+ * @param {string | null} params.frontToken - The frontToken containing issued timestamp.
+ * @param {string | null} params.responseHeaders - The headers for the response that returned the frontToken.
```

Comment:
The type here is wrong. Isn't there a way to omit it? I like the comment, but we already have the type because of TS.